client/package.json

```diff
@@ -53,6 +54,7 @@
     "webpack": "1.13.0"
   },
   "devDependencies": {
+    "@kadira/storybook": "^1.25.0",
```

Comment:
We've used exact version numbers in all the other packages, and should probably do the same here.